test/e2e/providers/vsphere/vsphere.go

```diff
@@ -110,17 +110,20 @@ func (p *Provider) getWorkspaceFromExistingMachineSet() (*mapi.Workspace, error)
 
 // getNetwork returns the network that needs to be used in the MachineSet
 func getNetwork() string {
-	// Default network for dev environment
-	networkSegment := "devqe-segment-222"
+	// $LEASED_RESOURCE holds the network the CI cluster is in and allows configuration of the network segment
+	// to use for testing with development clusters.
+	networkSegment := os.Getenv("LEASED_RESOURCE")
+	// check if the network segment is set and use it
+	if networkSegment != "" {
+		return networkSegment
+	}
+	// otherwise use default values
 	if os.Getenv("OPENSHIFT_CI") == "true" {
-		// $LEASED_RESOURCE holds the network the CI cluster is in
-		networkSegment = os.Getenv("LEASED_RESOURCE")
-		// Default to "ci-segment" if the environment variable is not set.
-		if networkSegment == "" {
-			networkSegment = "ci-segment"
-		}
+		// Default to "ci-segment" if the environment variable is not set while running in CI
+		return "ci-segment"
```

Comment:
Is this a valid segment to use anymore?

Reply:
IDK, and I have the same query. Anyhow, it's out of the scope here.